src/about.html

```diff
@@ -0,0 +1,287 @@
+	<script src="./js/frontPage.js" type="module" defer></script>
+</head>
+
+<body>
+	<header id="site-header" class="header py-3">
+		<div class="container-fluid gx-5">
+			<div class="row justify-content-between">
+				<div class="col-3 d-flex justify-content-around p-3">
+					<a href="/">Logo</a>
+					<div class="hamburger d-flex flex-column justify-content-evenly align-items-center">
+						<span class="hamburger__lines"></span>
+						<span class="hamburger__lines"></span>
+						<span class="hamburger__lines"></span>
+					</div>
+					<nav id="main-nav">
+						<ul>
+							<li>About Us</li>
+							<li>Services</li>
+							<li>Resources</li>
+							<li>Support</li>
+							<li>Store</li>
+							<li class="btn pill-btn">Get Started</li>
+						</ul>
+					</nav>
+				</div>
+				<div class="col-5 d-flex justify-content-center align-items-center">
+					<a href="#" class="header__social-icon mx-1">Facebook</a>
+					<a href="#" class="header__social-icon mx-1">Instagram</a>
+					<a href="#" class="header__social-icon mx-1">LinkedIn</a>
+					<div class="cta pill-btn mx-1">Get Started</div>
+				</div>
+			</div>
+		</div>
+	</header>
+	<main>
+		<section class="hero">
+			<div class="hero__background">
+				<div class="hero__background--lower"></div>
+				<div class="hero__background--upper"></div>
+			</div>
+			<div class="hero__content container-fluid d-flex align-items-stretch">
+				<div class="row">
+					<div class="col-3 d-flex flex-column">
+					</div>
+					<div class="col d-flex flex-column justify-content-center">
+						<div class="container">
+							<div class="hero__text">
+								<h1 class="hero__text--headline text-white leaf-adjustment"><img src="./assets/images/threeleaves.png">about us</h1>
+								<span class="hero__text--subheadline-1 subheadline white-stroke">WE EXIST TO BUILD COURAGEOUS, EFFECTIVE AND <br> HEALTHY MINISTRIES THAT GROW THE CHURCH.</span>
+								<div class="row">
+									<div class="col">
+										<a class="pill-btn__fill--primary" href="#">Get Started</a>
+									</div>
+								</div>
+							</div>
+						</div>s
+						
+					</div>
+				</div>
+			</div>
+		</section>
+		<section class="mission-slate">
+			<div class="container">
+				<div class="row">
+					<div class="col">
+						<h1 class="headline leaf-adjustment"><img src="/src/assets/images/greenthreeleaves.png">mission slate 23-24</h1>
+						<p>Zombie ipsum reversus ab viral inferno, nam rick Grimes malum cerebro. De <br> carne lumbering animata corpora quaeritis. Summus brains sit​​, morbo vel <br> maleficia? De apocalypsi gorger omero undead survivor dictum mauris. Hi<br>  mindless mortuis soulless creaturas, imo evil stalking monstra adventus resi <br> dentevil vultus comedat cerebella viventium.</p>
+					</div>
+					<div class="container mt-5 mb-5">
+						<div class="row">
+							<div class="col-lg-12">
+								<ul class="timeline">
+									<li>
+										<h1 class="headline">Quarter 1</h1>
```

Comment:
if you need multiple h1 styles, use an '.h1' class and a proper heading level in the html (e.g. `<h3 class='h1'>`)